package.json

```diff
@@ -18,15 +19,15 @@
   "author": "SubQuery Network",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@polkadot/api": "^8",
-    "@subql/types": "^1.1.0",
-    "@subql/cli": "^1.1.0",
+    "@polkadot/api": "^8.13.1",
+    "@polkadot/api-augment": "^8.13.1",
+    "@polkadot/types-augment": "^8.13.1",
+    "@polkadot/api-base": "^8.13.1",
+    "@subql/cli": "^1.6.1",
+    "@subql/types": "latest",
```

Comment:
Can we lock down the version here too?

Reply:
Sure 👍

Reply:
I've commented from other account. Got it @noisefloors